packages/mui-lab/src/LoadingButton/LoadingButton.js

```diff
@@ -179,8 +183,8 @@ const LoadingButton = React.forwardRef(function LoadingButton(inProps, ref) {
       classes={classes}
       ownerState={ownerState}
     >
-      {ownerState.loadingPosition === 'end' ? children : loadingButtonLoadingIndicator}
-      {ownerState.loadingPosition === 'end' ? loadingButtonLoadingIndicator : children}
+      {ownerState.loadingPosition === 'end' ? <div style={contentDivStyles}>{children}</div> : loadingButtonLoadingIndicator}
```

Comment:
We can't have a div here https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button 
```suggestion
      {ownerState.loadingPosition === 'end' ? <span style={contentDivStyles}>{children}</span> : loadingButtonLoadingIndicator}
```

Reply:
Also, please, no inline style, it's harder to customize and makes it harder to have strict CSP.

Reply:
I changed it to use a `span` and removed the inline styles.